Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "storage_device"
+version = "1.0.0"
+authors = ["Thog <me@thog.eu>", "orycterope <tvermeilh@gmail.com>"]
+keywords = ["storage", "block", "device", "block-device", "storage-device", "io"]
+categories = ["filesystem", "caching", "no-std"]
+description = "A storage device representation to help filesystem development."
+homepage = "https://github.com//sunriseos/storage_device"
+repository = "https://github.com//sunriseos/storage_device.git"
+edition = "2018"
+
+[package.metadata.docs.rs]
+features = ["cached-block-device"]
+
+[dependencies]
+lru = { git = "https://github.com/Orycterope/lru-rs", branch = "no_std", optional = true }
+spin = { version = "0.5.0", optional = true }
+
+[features]
+cached-block-device = ["lru", "spin"]
```

```suggestion
cached-block-device = ["lru/nightly", "spin"]
```

chached-block-device in nostd environment needs to enable lru's 'nightly' feature.

But this is not needed for when you're building with std.